nullaway/src/test/resources/com/uber/nullaway/testdata/CheckFieldInitPositiveCases.java

```diff
@@ -32,7 +32,7 @@
 
     Object f;
 
-    // BUG: Diagnostic contains: initializer method does not guarantee @NonNull field f is
+    // BUG: Diagnostic contains: initializer method does not guarantee @NonNull field f(Line:33) is
```

Comment:
This is going to make adding test cases to these files a bit harder. But: 1) we do need to test the new line number printing code too, so using existing test cases for it makes sense, 2) I am actually fine with discouraging this kind of test cases going forward vs proper unit tests via `.addLines(...)`, etc :)

p.s. nothing to change here, just a note.

Reply:
Yeah, I agree.  This will discourage us from messing with this older, worse way of writing tests in the future 😄

Reply:
Yes, thank you for pointing that out.